bugsnag-android-performance/src/main/kotlin/com/bugsnag/android/performance/internal/ImmutableConfig.kt

```diff
@@ -45,6 +47,7 @@ internal data class ImmutableConfig(
         configuration.networkRequestCallback,
         configuration.doNotEndAppStart,
         configuration.doNotAutoInstrument,
+        configuration.tracePropagationUrls,
```

Comment:
We need to take a defensive copy here to ensure that the `tracePropagationUrls` can't be mutated after startup:

```suggestion
        configuration.tracePropagationUrls.toSet(),
```